device/accton/x86_64-accton_as4630_54pe-r0/plugins/sfputil.py

```diff
@@ -49,7 +50,7 @@ def port_end(self):
 
     @property
     def qsfp_ports(self):
-        return range(self.PORT_START, self.PORTS_IN_BLOCK + 1)
+        return list(range(self.PORT_START, self.PORTS_IN_BLOCK + 1))
```

Comment:
what is the reason to add list() here?

Reply:
`qsfp_ports()` is expected to return a list. In Python 2, `range()` returned a list. However, it Python 3, it returns an "immutable sequence" which needs to be converted to a list.